homeworks/test17/task5/task5.py

```diff
@@ -0,0 +1,22 @@
+def palindrome_check(num):
+    try:
+        if num < 0:
+            return False
+        num_str = str(num)
+        return num_str == num_str[::-1]
+    except TypeError:
+        return False
```

Comment:
You can return `-1` as a wrong result, and log an error
```suggestion
    except TypeError as e:
        logging.error(e)
        return -1
```